src/math/web_mercator.rs

```diff
@@ -0,0 +1,143 @@
+//! Calculations with Web Mercator coordinates and tiles.
+
+use nalgebra::Vector2;
+use nav_types::WGS84;
+use std::f64::consts::{FRAC_1_PI, PI};
+
+/// 2.0 * E.powf(PI).arctan() - FRAC_PI_2;
+/// In degrees, it's 85.051129 (cf. Wikipedia)
```

Comment:
Nit: Can you please link to the URL instead of cf. Wikipedia?

Reply:
Done.